src/userLayout/Checkout.tsx

```diff
@@ -7,13 +7,19 @@ import {
 } from "@stripe/react-stripe-js";
 import { loadStripe } from "@stripe/stripe-js";
 import { FormEvent } from "react";
-import { axiosInstanceWithHeaders } from "../../axiosConfig/axiosInstance";
+import { axiosInstanceWithHeaders } from "../axiosConfig/axiosInstance";
+import { useLocation } from "react-router-dom";
+import { Button } from "@mui/material";
+import { toast } from "react-toastify";
 
 const stripePromise = loadStripe(
   "pk_test_51OTjURBQWp069pqTmqhKZHNNd3kMf9TTynJtLJQIJDOSYcGM7xz3DabzCzE7bTxvuYMY0IX96OHBjsysHEKIrwCK006Mu7mKw8"
 );
 
-const PayBooking = () => {
+const CheckOut = () => {
+  const location = useLocation();
+  const bookingId = location?.state?.bookingId;
```

Comment:
I  suggest that you store the id the URL instead. This will enable us to share the URL if we want to open in a different browser or share it with a different device.
```jsx
nivgate(`booking-path?bookingId=${bookindId}`)
```

and then we can use useSearchParams hook to get the parameter bookindId